src/pspm_convert_visangle2sps.m

```diff
@@ -32,13 +35,13 @@
 global settings;
 if isempty(settings), pspm_init; end;
 sts = -1;
+out = []

```

Comment:
This is the problem for the later issue ! (see below) You shouldn't initialize it or at least not like this. This forces `out` to be a numeric array and before it was creating a struct array.

Reply:
ah ok, great thanks for the heads up